NEWS

```diff
@@ -366,6 +366,11 @@ CHANGES WITH 258 in spe:
         * A new PrivateBPF= switch has been added for unit files, which may be
           used to mount a private bpffs instance for the unit's processes.
 
+        * Add four new options BPFDelegate{Commands,Maps,Programs,Attachments}=
```

Comment:
"Four new options added"

Reply:
Please do not use `{...}`, but list all options.